web/src/user_settings.ts

```diff
@@ -57,7 +57,7 @@ export type UserSettings = (StreamNotificationSettings &
     send_read_receipts: boolean;
 };
 
-export let user_settings = {} as UserSettings;
+export let user_settings: UserSettings;
```

Comment:
This uncovered a bug in #25759 where `browser_history` is trying to access `user_settings.default_view` before `user_settings.initialize_user_settings` is called: https://github.com/zulip/zulip/pull/25759#discussion_r1320422274.